source/draw/gpu/opengl/AndroidContext.ooc

```diff
@@ -165,6 +165,11 @@ AndroidContext: class extends OpenGLContext {
 		}
 		result
 	}
-	recycleEGLRgba: func (image: EGLRgba) { this _eglBin add(image) }
```

Comment:
This change is already up in a PR: https://github.com/magic-lang/ooc-kean/pull/1895